labs/build-your-first-gadget/README.md

```diff
@@ -182,7 +177,7 @@ To decipher the function definition: The first macro parameter is the syscall na
 ```C
 SYSCALL_DEFINE3(open, const char __user *, filename, int, flags, umode_t, mode)
 ```
-and therefore the filename should be found in `args[0]` inside the `struct syscall_trace_enter`. We also need to cast the parameter to `const char *`.
+and therefore the filename should be found in `args[0]` inside the `struct syscall_trace_enter`. For `openat` the filename should be found in `args[1]`. We also need to cast those parameters to `const char *`.
```

Comment:
I think when writing this I intentionally didn't mentioned that for people to figure out

```suggestion
and therefore the filename should be found in `args[0]` inside the `struct syscall_trace_enter`. We also need to cast those parameters to `const char *`.
```